aws-s3-transfer-manager/src/operation/upload/service.rs

```diff
@@ -25,6 +24,21 @@ pub(super) struct UploadPartRequest {
     pub(super) upload_id: String,
 }
 
+impl Policy<UploadPartRequest> for hedge::DefaultPolicy {
```

Comment:
Maybe add a simple unit test for the policy or otherwise test that we aren't breaking s3 express.

Reply:
I am not sure what kind of unit test we will need here. we don't have tests for hedge at all.
I ran some benchmark manually and prove it works. 
I guess it will fall into the integration tests we are missing.

I could add some test to check the s3 express was detected correctly?

Reply:
Yeah a simple unit test of the policy that checks `clone_request` for express and non-express buckets. The logic is so simple it seems unnecessary but I think it serves more for future us if we refactor the logic or change the policy we have a simple test to keep us honest and not regress.